src/wwwroot/template-editor/adapterService.js

```diff
@@ -340,7 +352,7 @@
     function getSettings(idCampaign, idTemplate) {
       traceAdapterCall(getSettings, arguments);
       var msEditorSettings = {
-        language: 'es',
+        language: langKey,
         sharedSocialNetworks: [{
```

Comment:
Since by the moment we have all that we need from this method, could you finish it?

I think that it is only pending to:

* removing sharedSocialNetworks
* removing stores
* make rss related properties _false_
* set _redirectUrl_ with a Doppler Relay URL

Reply:
You could also remove the line related to _traceAdapterCall_, but I am not sure, we can do that later.

Reply:
Of course.